flow/workflows/drop_flow.go

```diff
@@ -85,7 +86,12 @@ func DropFlowWorkflow(ctx workflow.Context, input *protos.DropFlowInput) error {
 	ctx = workflow.WithValue(ctx, shared.FlowNameKey, input.FlowJobName)
 	workflow.GetLogger(ctx).Info("performing cleanup for flow",
 		slog.String(string(shared.FlowNameKey), input.FlowJobName))
-
+	newCtx, err := GetFlowMetadataContext(ctx,
+		input.FlowJobName, input.FlowConnectionConfigs.SourceName, input.FlowConnectionConfigs.DestinationName)
+	if err != nil {
+		return fmt.Errorf("failed to get flow metadata context: %w", err)
+	}
+	ctx = newCtx
```

Comment:
```suggestion
	var err error
	ctx, err = GetFlowMetadataContext(ctx,
		input.FlowJobName, input.FlowConnectionConfigs.SourceName, input.FlowConnectionConfigs.DestinationName)
	if err != nil {
		return fmt.Errorf("failed to get flow metadata context: %w", err)
	}
```

Reply:
makes sense, ctx is not needed once we have an error